### DISCUSSION TITLE
Github issue in command git push using LFS tools

### DISCUSSION THREAD

**Discussion by pranit-elitecontrivers** [community/community#102867] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have an issue with git push command where it shows this exceeds GitHub's file size limit of 100.00 MB as shown in below screenshot.
![git-lfs](https://github.com/community/community/assets/138686685/f9236c57-5902-4ecd-8193-5a6bf4eb8192)

**Comment by phuccoder2000** [comment]
Hi @pranit-elitecontrivers 
1. Git LFS (Large File Storage):
  - If the file is a binary or large asset that doesn't change often, consider using Git LFS (Large File Storage). Git LFS allows you to store large files outside the Git repository, reducing the size of your Git repository.
2. Git Push with --force:
 - If you're certain that you want to push the large file and overwrite history, you can use git push --force. Be cautious with this, as it can rewrite history and affect collaborators.
3. Remove Large Files:
  - If the large file is not necessary for your repository, you might want to remove it from version control and use other means to distribute or store it.
